packages/Toaster.vue

```diff
@@ -358,16 +358,16 @@ watchEffect((onInvalidate) => {
       (key) => (event as any)[key] || event.code === key
     )
 
+    const listRefItem = Array.isArray(listRef.value) ? listRef.value.at(0) : listRef.value
```

Comment:
It seems like the `.at` syntax cannot be used in the repo. Can we use `listRef.value[0]` instead?

Reply:
Sure, no problem.